src/Form/Field/ScopeBuilder.php

```diff
@@ -0,0 +1,611 @@
+<?php
```

Comment:
@georgehristov with there PRs, I have to be able to rely on you that these mistakes are not present

Reply:
@mvorisek that is why we have the review process ...

Reply:
yes, but you should optimize your workflow to prevent such mistakes - if you do it cleanly, it can not happen